api/src/cli/commands/security/key.ts

```diff
@@ -0,0 +1,30 @@
+import { readFileSync, existsSync, writeFileSync } from 'fs';
+import { join as pathJoin } from 'path';
+import { v4 as uuidv4 } from 'uuid';
+import logger from '../../../logger';
+
+export default async function generateKey({ show }: { show?: boolean }): Promise<void> {
+	const key = uuidv4();
+
+	if (show) {
+		process.stdout.write(`Your new app key: ${key}\n`);
+
+		process.exit(0);
+	}
+
+	const rootPath = process.cwd();
+	const envPath = pathJoin(rootPath, '.env');
```

Comment:
The path where the config file resides is user configurable, so we can't rely on it being the root + '.env'